README.md

```diff
@@ -1,151 +1,119 @@
-- [wdl-packager](https://github.com/biowdl/wdl-packager) packages a workflow and all of its imports into a zip file. The zip can be used as an imports zip package for cromwell. The utility can add non-WDL files (such as the license) to the zip package and provides options to package the zip in a binary reproducible way.
-
-## Contributing
-
-WDL only advances through community contributions.
-While participating in discussions and submitting issues are great ways to be involved, help is also needed to implement changes to the specification.
-For more information on how you can contribute, please read the [Contributing](CONTRIBUTING.md) guide.
-
-### RFC Process
-
-Submitted [pull requests](https://github.com/openwdl/wdl/pulls) are subject to the [RFC Process](RFC.md).
-Please review and familiarize yourself with the process if you would like to see changes submitted to the specification.
-
-## Governance
-
-The WDL specification is entirely community driven; however, it is overseen by a [Governance committee](GOVERNANCE.md).
-If you are interested in being involved in WDL governance, please join the [Slack](https://join.slack.com/t/openwdl/shared_invite/zt-ctmj4mhf-cFBNxIiZYs6SY9HgM9UAVw) and post a message in the `#general` channel.
+<div style="align: center">
+  <img src="https://raw.githubusercontent.com/claymcleod/wdl/refs/heads/docs/readme-update/.github/assets/readme-header.png">
+</div>
+
+<br />
+
+The **Workflow Description Language (WDL)** (pronounced as _/hwɪdl/_ or "whittle" with a
+'d') is an open standard for describing data processing workflows using a
+human-readable/writeable syntax. It introduces a domain-specific language that aims to
+provide simple but powerful facilities for (a) defining atomic units of computation (a
+`task`), (b) for connecting those atomic units together into a larger computation graph (a
+`workflow`), and (c) for effortlessly scaling the execution of these graphs in multiple
+environments.
+
+Most fundamentally, WDL differentiates itself from other workflow languages by focusing
+on ease of implementation and accessibility to wide range of users, including software
+engineers, domain experts (e.g, biologists), and operators of production computational
+systems. Furthermore, idiomatic workflow execution patterns, such as conditional
+execution, dynamic resource allocation, and scatter-gather operations, are simply
+expressed. Last, the language is driven and developed by community of distributed
+volunteers as an open standard—there is no single execution engine, and anyone can build
+tooling to contribute to the ecosystem. Today, [several engines and
+platforms](#execution-engines-and-platforms) exist and cover most execution environments
+(all major HPCs, all major clouds, local execution).
+
+## Getting Started
+
+If you're new to WDL, welcome 👋—we _highly_ recommend you read the [getting started
+documentation][wdl-docs] to learn how to write workflows. Beyond that, here are some
```

Comment:
```suggestion
documentation][wdl-docs] to learn how to write tasks and workflows. Beyond that, here are some
```

Reply:
I don't think the sentence starting "Beyond that..." is necessary as there's really only one sub-section ("Join the community"). I also don't think the "Conclusion" section is necessary.